daemon/algod/api/server/v2/handlers.go

```diff
@@ -85,6 +87,8 @@ type LedgerForAPI interface {
 	EncodedBlockCert(rnd basics.Round) (blk []byte, cert []byte, err error)
 	Block(rnd basics.Round) (blk bookkeeping.Block, err error)
 	AddressTxns(id basics.Address, r basics.Round) ([]transactions.SignedTxnWithAD, error)
+	GetAccountDeltasForRound(rnd basics.Round) (ledgercore.AccountDeltas, error)
+	GetKvDeltasForRound(rnd basics.Round) (map[string]ledgercore.KvValueDelta, error)
```

Comment:
I think I'd rather we expose the `ledgercore.StateDelta`.  It has already been designed to convey all of the changes from one round to the next (in fact, even from one txn or txgroup to the next, as it is also the `mods` datastructure in a `rowCowBase`). So it would be natural to presume anything in it might be valuable for an `indexer`-like client to accumulate. (Leases taken, txids used)

I wouldn't be opposed to something that let's clients decide what parts they want, but I would like to avoid a whole new code path for each thing (GetAccounts..., GetKvDeltas..., GetLeases...)